docs/integrations-guides/rollup-guides/orbit/deployment.md

```diff
@@ -34,32 +34,9 @@ yarn hardhat run scripts/local-deployment/deployCreator.ts
 
 The script will take a few minutes to complete as it prints out the addresses of the deployed contracts along the way. Lastly, the script will write the deployed contracts to the `CREATOR_DEPLOYMENT_INFO` file you specified. Upon completion, your rollup creator factory is ready to use to deploy new chains. 
 
-## How to deploy a Rollup using pre-deployed Rollup Creators
```

Comment:
Did you mean to delete all this? Seems unrelated to pr title

Reply:
yeah this refers to scripts that we don't support anymore